protocols/index/contracts/Index.sol

```diff
@@ -13,207 +13,196 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */
-
 pragma solidity 0.5.12;
 pragma experimental ABIEncoderV2;
 
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
 /**
-  * @title Index: A List of Entries
+  * @title Index: A List of Locators
   */
 contract Index is Ownable {
 
-  // Length of the linked list
+  // Number of entries in the index
   uint256 public length;
 
-  // Maximum address value to indicate the head
+  // Identifier to use for the head
   address private constant HEAD = address(uint160(2**160-1));
 
-  // Mapping of user address to its neighbors
-  mapping(address => Entry) private _listEntries;
+  // Mapping of identifier to its entry
+  mapping(address => Entry) private _entries;
 
   /**
-    * @notice Entry for a locator
-    * @dev locator is arbitrary e.g. may include an address
-    * @param user address
+    * @notice Index Entry
     * @param score uint256
     * @param locator bytes32
+    * @param prev address
+    * @param next address
     */
   struct Entry {
+    bytes32 locator;
+    uint256 score;
     address prev;
     address next;
-    uint256 score;
-    bytes32 locator;
   }
 
   /**
     * @notice Contract Events
-    * @dev Emitted with successful state changes
     */
-
-  event SetEntry(
+  event SetLocator(
+    address indexed identifier,
     uint256 score,
-    address indexed user,
     bytes32 indexed locator
   );
 
-  event UnsetEntry(
-    address indexed user
+  event UnsetLocator(
+    address indexed identifier
   );
 
   /**
     * @notice Contract Constructor
     */
   constructor() public {
-    // Initialize the linked list.
-    _listEntries[HEAD] = Entry(HEAD, HEAD, 0, bytes32(0));
+    // Create initial entry.
+    _entries[HEAD] = Entry(bytes32(0), 0, HEAD, HEAD);
   }
 
   /**
-    * @notice Set an Entry
-    *
-    * @param _user The account
+    * @notice Set a Locator
+    * @param _identifier address
     * @param _score uint256
     * @param _locator bytes32
     */
-  function setEntry(
-    address _user,
+  function setLocator(
+    address _identifier,
     uint256 _score,
     bytes32 _locator
   ) external onlyOwner {
 
-    require(!hasEntry(_user), "ENTRY_ALREADY_EXISTS");
+    // Ensure the entry does not already exist.
+    require(!hasEntry(_identifier), "ENTRY_ALREADY_EXISTS");
 
-    // Find the first user who has a lower stake, and insert before them
-    address nextUser = findPosition(_score);
+    // Find the first entry with a lower score.
+    address nextEntry = getEntryLowerThan(_score);
```

Comment:
YES GREAT RENAMING. I've hated `findPosition` for so long.
![image](https://user-images.githubusercontent.com/34962750/67579144-e3cedb00-f711-11e9-8298-7a417d652742.png)